models/setting.js

```diff
@@ -0,0 +1,14 @@
+module.exports = (sequelize, DataTypes) =>
+  sequelize.define('Setting', {
+    key: {
+      type: DataTypes.TEXT,
```

Comment:
I forget, but is there a string type?  I'm wondering if text was for really long strings.